.github/workflows/check-query-files-and-compilation.yml

```diff
@@ -56,15 +56,29 @@ jobs:
         run: |
           bash ./scripts/ci-install-${{ matrix.os }}.sh
 
+      - name: Setup Parsers Cache
+        id: parsers-cache
+        uses: actions/cache@v2
+        with:
+          path: ./parser/
+          key: ${{ matrix.os }}-parsers-v1-${{ hashFiles('./lockfile.json', './lua/nvim-treesitter/parsers.lua', './lua/nvim-treesitter/install.lua', './lua/nvim-treesitter/shell_selectors.lua') }}
+
+      - name: ccache
+        if: matrix.os != 'windows-2022'
+        uses: hendrikmuhs/ccache-action@v1
+        with:
+          key: ${{ matrix.os }}-${{ matrix.cc }}
```

Comment:
Don't really know about what this cache will contain, but it seems like this will cache it forever? Won't it grow in size over-time?

Reply:
no, it's a 500MB ccache size

Reply:
no really a benefit, as extracting takes longer than compiling.